nilearn/plotting/img_plotting.py

```diff
@@ -383,6 +383,17 @@ def _load_anat(anat_img=MNI152TEMPLATE, dim='auto', black_bg='auto'):
             else:
                 black_bg = True
     if dim:
+        if dim != 'auto' and not isinstance(dim, numbers.Number):
+            raise ValueError(
```

Comment:
Yes, but from reading the code, I think that we can accept values between -2 and +2.